src/AspNet.Security.OAuth.Strava/StravaAuthenticationHandler.cs

```diff
@@ -1,9 +1,11 @@
-/*
+﻿/*
  * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
  * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
  * for more information concerning the license and the contributors participating to this project.
  */
 
+using System;
+using System.Linq;
```

Comment:
`StringValues` implements `IList<string>`, so you can just use `[0]` rather than `First()`, so you don't need this `using`.